portfolio/src/main/webapp/script.js

```diff
@@ -34,7 +35,37 @@ function getComments() {
             commentsContainer.appendChild(commentContent);
         }
     });
+}
 
+/**
+* Converts timeStamp from UTC to local timezone
+* @param {string} timeStamp string in form mm/dd/yyyy hh:mm a
+* @return {string} string in form mm/dd/yyyy hh:mm a for local timezone
+*/
+function convertTimeStamp(timeStamp) {
+    const timeStampString = timeStamp + " UTC";
```

Comment:
This doesn't seem very robust. My recommendation is to store the timestamp initially as a timestamp object instead of a string in DataServlet.java and using something similar to var localDate = new Date(utcDate); to set it here?

Reply:
Thanks for the suggestion! I have attempted to address this in the most recent commits.